src/model/stl/vector_impl.rs

```diff
@@ -1,10 +1,11 @@
-use std::ops::{Deref, DerefMut, Index, IndexMut};
```

Comment:
rustfmt changes

Reply:
These changes come from building with the `generate_model_apis` feature on.